fim_pipeline.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash -e
+
+: '
+fim_pipeline.sh -u <huc8> -n <name_your_run>
+
+For more details on 
+
+- There are a wide number of options and defaulted values, for details run ```gms_pipeline.sh -h```
+- Manditory arguments:
+    - `-u` can be a single huc, a series passed in quotes space delimited, or a line-delimited file
+    i. To run entire domain of available data use one of the ```/data/inputs/included_huc[4,6,8].lst``` files or a huc list file of your choice.
+    - `-n` is a name of your run (only alphanumeric)
+- Outputs can be found under ```/data/outputs/<name_your_run>```
+
+Processing of HUCs in FIM4 (GMS) comes in two pieces: gms_run_unit and gms_run_branch. `gms_pipeline.sh` above takes care of both steps however, you can run each part seperately for faster development if you like.
```

Comment:
Outdated references here.

Reply:
Fixed.. .thank you (coming in next push)